src/styles/pages/services/services.module.scss

```diff
@@ -0,0 +1,178 @@
+.mainContainer {
+    font-family: Arial; //import later 
+    display: flex;
+    flex-direction: column;
+}
+
+.servicesContainer {
+    background-color: var(--color-white);
+    color: var(--color-maroon);
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 32px;
+    row-gap: 40px;
+    justify-content: space-around;
+
+    .pageHeading {
+        font-size: 3.4375rem;
+        //font-family: "Be Vietnam";
```

Comment:
To resolve the header being out of place, you can add `text-align: center` into the `.pageHeading` selector